cpp/ql/src/semmle/code/cpp/ssa/internal/aliased_ssa/IRBlock.qll

```diff
@@ -1,72 +1,31 @@
 private import IRInternal
+private import Construction::OldIR as OldIR

```

Comment:
Maybe it's not optimal to refer to the previous IR here. We could instead refer all the way back to the first IR and skip the intermediate step in the case of `aliased_ssa`.